L1Trigger/TrackFindingTracklet/python/Tracklet_cfi.py

```diff
@@ -25,6 +26,10 @@
                                                TrackQualityPSet = cms.PSet(TrackQualityParams)
     )
 
+TrackletTracksFromTrackletEmulation = TTTracksFromTrackletEmulation.clone(
```

Comment:
Add comment saying this is to run Tracklet pattern reco with new KF.

Reply:
done.